compute/instances/how-to/migrate-routed-ips.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: How to migrate an Instance to routed flexible IPs
+  description: This page explains how to migrate your Instance to routed flexible IPs
+content:
+  h1: How to migrate an Instance to routed flexible IPs
+  paragraph: This page explains how to hmigrate your Instance to routed flexible IPs
+tags: instance flexible ip routed
+dates:
+  validation: 2023-10-11
+  posted: 2023-10-11
+categories:
+  - compute
+---
+
+During deployment, each Instance gets a private IP from its hosting node, besides its public IP that is routed to the Instance through a carrier-grate NAT solution. However, if the underlying hypervisor changes, the private IP changes. This can happen when you restart your Instance. This private network lets Instances communicate without public IPs. 
```

Comment:
```suggestion
During deployment, each Instance gets a private IP from its hosting node, besides its public IP, which is routed to the Instance through a carrier-grate NAT solution. However, if the underlying hypervisor changes, the private IP changes. This can happen when you restart your Instance. This private network lets Instances communicate without public IPs. 
```